source/isaaclab/isaaclab/sensors/tacsl_sensor/visuotactile_sensor_data.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022-2025, The Isaac Lab Project Developers (https://github.com/isaac-sim/IsaacLab/blob/main/CONTRIBUTORS.md).
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+from __future__ import annotations
+
+import torch
+from dataclasses import dataclass
+
+
+@dataclass
+class VisuoTactileSensorData:
+    """Data container for the visuo-tactile sensor.
+
+    This class contains the tactile sensor data that includes:
+    - Camera-based tactile sensing (RGB and depth images)
+    - Force field tactile sensing (normal and shear forces)
+    - Tactile point positions and contact information
+    """
+
+    # Camera-based tactile data
+    tactile_camera_depth: torch.Tensor | None = None
+    """Tactile depth images. Shape: (num_instances, height, width)."""
+
+    tactile_rgb_image: torch.Tensor | None = None
+    """Tactile RGB images rendered using the Taxim approach (https://arxiv.org/abs/2109.04027). Shape: (num_instances, height, width, 3)."""
```

Comment:
Would make sense to follow same names for these two. Also I think shape should be consistent for both images?